webapp/public/locales/fr/main.json

```diff
@@ -130,7 +130,7 @@
   "tasks.LAUNCH": "Lancement",
```

Comment:
Use ' instead of `

Reply:
@skamril, tu veux que je remplace les apostrophes ([U+2019](https://www.compart.com/fr/unicode/U+2019)) par des quotes droites ([U+0027](https://www.compart.com/fr/unicode/U+0027)), c'est bien ça ?
En français, on utilise l'apostrophe, donc je préfère les garder comme ça. Mais par de souci d'homogénéité, je suis prêt à changer (mais je n'aime pas trop ça) 😁.